### PR TITLE
Font awesome and nav bar tweaks

### PR DESCRIPTION
**Pull request by RafaelCenzano**
Including adding secret.py for keys

### CODE REVIEW

**Comment by RafaelCenzano** [comment]
small changes still need to tweak colors and spacing of icons

File: lowell-dashboard/app/config.py [hunk 1]
```diff
@@ -3,13 +3,15 @@
 Ex: database configurations, keys, etc.
 '''
 
+import secret
```
Comment (calee14) [inline lowell-dashboard/app/config.py:6]: i think it has to be from app import secret since it's in the app folder.

**Review by calee14 — CHANGES_REQUESTED** [review]

File: lowell-dashboard/app/config.py (hunk 1 above)
Comment (RafaelCenzano) [reply lowell-dashboard/app/config.py:6]: In the same folder as config folder

**Comment by RafaelCenzano** [comment]
Make more tweaks to Navbar. Waiting to fix login/register button and settings/logout button once jinja has the function to check if the user is anonymous or logged in.

**Review by calee14 — APPROVED** [review]

**Pull request merged**